### PR TITLE
docs: add issue tracking process

### PR DESCRIPTION
**Pull request by BbrSofiane**

### CODE REVIEW

File: docs/decisions/0004-issue-tracking.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+the BTR group on Github. These guidelines that are expected to evolve 
+to meet any particular need that arises.
+
+Last update: 2021-03-15
+
+---------
+Decisions
+---------
+
+#. A piece of work should be created as a github issue. Github projects <support notes",
```
Comment (arbrandes) [inline docs/decisions/0004-issue-tracking.rst:28]: Typo: "<support notes" - supports notes

File: docs/decisions/0004-issue-tracking.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+#. A piece of work should be created as a github issue. Github projects <support notes", 
+   however these are too basic and don't support the additional information issues allows 
+   (for example, an assignee).
+
+#. To assess and discuss each issue, we need to know:
+
+   #. The type of issue: is it a bug? is it a gap in documentation?
+        - Tagged with <type:'issue-type'>
+        - Issue types: bug, enhancement, question, btr-task, security.
```
Comment (arbrandes) [inline docs/decisions/0004-issue-tracking.rst:36]: Why do we need a "question" type?

Also, I think "btr-task" is a bit too vague.  What idea is it meant to convey that "bug" or "enhancement" don't?

File: docs/decisions/0004-issue-tracking.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+   #. The release that is affected. 
+        - Tagged with <affects:'named-released'>
```
Comment (arbrandes) [inline docs/decisions/0004-issue-tracking.rst:38]: I'm for this, but I don't think *every* issue needs an "affects:release".  For instance, an "enhancement" can't really affect anything until it exists.

I actually think that "affects" should be used almost exclusively for bugs.  Everything else should just have a milestone.

**Review by arbrandes — COMMENTED** [review]
@BbrSofiane, I think this looks great, thanks!  I left some line comments.

Also, I think this is missing something: can you please add wording explaining that the Triager will move tickets from Incoming to To Do once the group decides that something will actually be done, and to Not Doing if not?  This can be added to the line that explains setting a milestone.  (In other words, any tickets in the To Do column must have a milestone set.)

Finally, I think we should target merging this by March 29, if there are no objections.

File: docs/decisions/0004-issue-tracking.rst (hunk 2 above)
Comment (BbrSofiane) [reply docs/decisions/0004-issue-tracking.rst:36]: btr-task to put things like: "create koa.3 tags" or any admin task related to the group.

File: docs/decisions/0004-issue-tracking.rst (hunk 3 above)
Comment (BbrSofiane) [reply docs/decisions/0004-issue-tracking.rst:38]: Yes, it makes sense.

**Review by arbrandes — APPROVED** [review]
LGTM!

**Pull request merged**